src/components/CurrentEpochTime.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactElement, useEffect, useRef, useState } from 'react';
+
+import { CopyIcon } from './icons/CopyIcon';
+import { OkIcon } from './icons/OkIcon';
+
+function useInterval(callback: () => void, delay: number): void {
+  const savedCallback = useRef<() => void>();
+
+  useEffect(() => {
+    savedCallback.current = callback;
+  }, [callback]);
+
+  useEffect(() => {
+    function tick(): void {
+      if (savedCallback.current) {
+        savedCallback.current();
+      }
+    }
+    if (delay) {
+      const id = setInterval(tick, delay);
+      return () => clearInterval(id);
+    }
+  }, [delay]);
+}
```

Comment:
Do I get it right, that this is better than normal `setInterval`, because it doesn't fire after component unmount?